modules/form/AboutYourChild.js

```diff
@@ -1,36 +1,142 @@
 import React from 'react'
 
+import FormRow from './FormRow.js'
+import TextInput from './inputs/TextInput'
+import SmallTextInput from './inputs/SmallTextInput'
+import NameInput from './inputs/NameInput'
+import GenderSelect from './inputs/GenderSelect'
+import BirthdateSelect from './inputs/BirthdateSelect'
+import AgeInput from './inputs/AgeInput'
+import PhoneNumberInput from './inputs/PhoneNumberInput'
+import SocialSecurityInput from './inputs/SocialSecurityInput'
+import AddressInput from './inputs/AddressInput'
+
 export default React.createClass({
-  saveAndContinue: function(e){
-    e.preventDefault();
+  getInitialState: function() {
+    console.log(this.props);
+    return {
+      data: {}
+    }
+  },
+
+  checkForKey: function(key) {
+    var data = this.state.data;
+    if (!data[key]) {
+      data[key] = {};
+    }
+  },
 
-    var data = {
-      firstName: this.refs.firstName.value,
-      lastName: this.refs.lastName.value
-    };
+  setBirthDateInfo: function(key, value) {
+    this.checkForKey('birthDate');
+    this.state.data.birthDate[key] = value;
+  },
+
+  setHomePhoneInfo: function(key, value) {
+    this.checkForKey('homePhone');
+    this.state.data.homePhone[key] = value;
+  },
 
-    this.props.saveValues(data);
+  setSocialSecurityInfo: function(key, value) {
+    this.checkForKey('socialSecurity');
+    this.state.data.socialSecurity[key] = value;
+  },
+
+  setAddressInfo: function(key, value) {
+    this.checkForKey('address');
+    this.state.data.address[key] = value;
+  },
+
+  saveAndContinue: function(e){
+    e.preventDefault();
+    this.props.saveValues(this.state.data);
     this.props.nextStep();
   },
 
   render: function(){
     return (
       <div className="container main form">
-        <div className="row">
-          <div className="col-lg-12">
-            <h1>About Your Child</h1>
 
-            <input type="text" ref="firstName" defaultValue={this.props.fieldValues.firstName} placeholder="Enter First Name"/>
+      <FormRow label="Patient's Name">
+        <NameInput 
+            firstName={this.props.fieldValues.firstName}
+            middleInitial={this.props.fieldValues.middleInitial}
+            lastName={this.props.fieldValues.lastName}
```

Comment:
So this was the first component I made LOL I can probably make it similar to `BirthDateSelect`, `SocialSecurityInput`, etc. and have a `patientName` object to store in the data and pass in for the defaults!